src/Compilers/CSharp/Test/Syntax/Parsing/MemberDeclarationParsingTests.cs

```diff
@@ -9107,5 +9107,107 @@ public void ReadonlyParameter4()
+                        N(SyntaxKind.PredefinedType);
+                        {
+                            N(SyntaxKind.IntKeyword);
+                        }
+                        N(SyntaxKind.IdentifierToken, "MyProperty");
+                        N(SyntaxKind.AccessorList);
+                        {
+                            N(SyntaxKind.OpenBraceToken);
+                            N(SyntaxKind.GetAccessorDeclaration);
+                            {
+                                N(SyntaxKind.GetKeyword);
+                                N(SyntaxKind.SemicolonToken);
+                            }
+                            N(SyntaxKind.SetAccessorDeclaration);
+                            {
+                                N(SyntaxKind.SetKeyword);
+                                N(SyntaxKind.SemicolonToken);
+                            }
+                            N(SyntaxKind.CloseBraceToken);
+                        }
+                    }
+                    N(SyntaxKind.CloseBraceToken);
+                }
+                N(SyntaxKind.EndOfFileToken);
+            }
+            EOF();
+        }
+
+        [Fact, WorkItem(52, "https://github.com/dotnet/roslyn/issues/52")]
+        public void PropertyWithErrantSemicolon2()
+        {
+            var text = @"
+public class Class
+{
+    public int MyProperty; => 0;
+
+    // Pretty much anything here causes an error
+}
+";
+            UsingTree(text,
+                // (4,26): error CS1514: { expected
+                //     public int MyProperty; => 0;
+                Diagnostic(ErrorCode.ERR_LbraceExpected, ";").WithLocation(4, 26));
+            N(SyntaxKind.CompilationUnit);
+            {
+                N(SyntaxKind.ClassDeclaration);
+                {
+                    N(SyntaxKind.PublicKeyword);
+                    N(SyntaxKind.ClassKeyword);
+                    N(SyntaxKind.IdentifierToken, "Class");
+                    N(SyntaxKind.OpenBraceToken);
+                    N(SyntaxKind.PropertyDeclaration);
+                    {
+                        N(SyntaxKind.PublicKeyword);
+                        N(SyntaxKind.PredefinedType);
+                        {
+                            N(SyntaxKind.IntKeyword);
+                        }
+                        N(SyntaxKind.IdentifierToken, "MyProperty");
+                        N(SyntaxKind.ArrowExpressionClause);
+                        {
+                            N(SyntaxKind.EqualsGreaterThanToken);
+                            N(SyntaxKind.NumericLiteralExpression);
+                            {
+                                N(SyntaxKind.NumericLiteralToken, "0");
+                            }
+                        }
+                        N(SyntaxKind.SemicolonToken);
+                    }
+                    N(SyntaxKind.CloseBraceToken);
+                }
+                N(SyntaxKind.EndOfFileToken);
+            }
+            EOF();
+        }
```

Comment:
> } [](http://example.com/codeflow?start=7&length=2)

Are we testing script cases for global statements?

Reply:
we are, yes.  and they def discovered interesting stuff for example, that `int MyProperty; {` is totally legal at the script/top-level as a local/field, followed by a block :)